Hatten_CletusPA1.asm

```diff
@@ -0,0 +1,90 @@
+.data
+	prompt: 	.asciiz "How many numbers do you want to input? "
+	message: 	.asciiz "\nThe number of inputs you have chosen is "
+	spaceHere:	.asciiz "\n"
+	
+	fibs:   	.word 0:9
+	sizelimit: 	.word 9
+
+.text
+	li $v0, 4		#Prompt user for input
+	la $a0, prompt
+	syscall 
+	li $v0, 5		#Stores user's input
+	syscall
+	move $t0, $v0		#Stores user's input inside temporary file
+	li $v0, 4		#Displays message
+	la $a0, message
+	syscall
+	li $v0, 1		#Displays user's input
+	move $a0, $t0		#Stores $t0 into an argument
```

Comment:
-1 No need to move $t0 to $s0 through $a0, its position in either $t0 or $s0 is ok, but no need for all the moving.